### PR TITLE
Updated protocol to 1.13.1

### PR DESCRIPTION
**Pull request by lluiscab**
See #203 for protocol changes.

### CODE REVIEW

**Comment by rom1504** [comment]
Could you rebase this into 2 commits with the first one being the copy of the 1.13 protocol ? Otherwise it's not possible to review this on GitHub...

File: data/pc/common/protocolVersions.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "minecraftVersion": "1.13.1",
+    "version": 401,
+    "usesNetty": true,
+    "majorVersion": "1.13.1"
+  },
```
Comment (rom1504) [inline data/pc/common/protocolVersions.json]: this file must be exhaustive, please complete with https://wiki.vg/Protocol_version_numbers

**Comment by rom1504** [comment]
Yes

**Comment by lluiscab** [comment]
Okay, I'll do that asap

**Comment by lluiscab** [comment]
Missing snaphosts have been added

**Pull request merged**